src/test/java/org/jenkinsci/plugins/scriptler/ScriptlerManagementTest.java

```diff
@@ -0,0 +1,46 @@
+package org.jenkinsci.plugins.scriptler;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.markup.MarkupFormatter;
+import hudson.markup.RawHtmlMarkupFormatter;
+import java.io.IOException;
+import java.io.Writer;
+import org.junit.jupiter.api.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.junit.jupiter.WithJenkins;
+
+@WithJenkins
+class ScriptlerManagementTest {
+
+    @Test
+    void markupFormatter(@SuppressWarnings("unused") JenkinsRule r) throws IOException {
```

Comment:
Why do we need to suppress an unused warning here? `r` is used.

Reply:
Good catch, copy&paste failed me here